styles.css

```diff
@@ -0,0 +1,7 @@
+* {
+  padding: 0;
+}
+
+.header {
+  color: red;
+}
```

Comment:
### Your Linters :red_circle:

![Screenshot from 2022-09-27 18-58-30](https://user-images.githubusercontent.com/93039359/192548556-e63a4595-18c9-495a-a8cf-9ce9d2160666.png)


- [ ] Linters are not passing all the tests. Kindly fix these errors. :+1: If you want to fix them automatically then run the following commands in your vscode code terminal :arrow_down:

```
npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x

npx stylelint "**/*.{css,scss}" --fix
```